src/sync/build.test.ts

```diff
@@ -39,6 +39,7 @@ describe("sync/builds", () => {
 	};
 
 	let repoSyncState: RepoSyncState;
+	const ORIGINAL_BUILDS_CURSOR = 21;
```

Comment:
What's the significance of 21?

Reply:
Just a random number @rachellerathbone , the position of the builds cursor for the tests